### PR TITLE
feat(docs): add Pyth Oracle Guide

### PR DESCRIPTION
**Pull request by mariobern**
## Description 

Hi all, Mario from Pyth here. Would like to add Pyth oracle and how to use Pyth on Sui for developers.

### CODE REVIEW

**Comment by mariobern** [comment]
Hi @siomari @ronny-mysten would love to get a review on this PR. Also seems like the vercel build has failed, but I couldn't check for the error.

Is there something that I need to do or can you help from your end?

File: docs/content/guides/developer/app-examples/price-oracle.mdx [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+     ```bash title='yarn'
+    yarn add @pythnetwork/pyth-sui-js
+    ```
+
+
+## Write Contract Code
+
+The code snippet below provides a general template for what your contract code should look like:
+
+```rust
```
Comment (damirka) [inline docs/content/guides/developer/app-examples/price-oracle.mdx:75]: Can you please make sure the code follows Move 2024 styles? Utilising module label, calling methods on structs, and following convention in error naming?

```move
module pyth_example::main;

use sui::clock::Clock;

// ...
const EInvalidId: u64 = 1;
```

Comment (damirka) [reply docs/content/guides/developer/app-examples/price-oracle.mdx:75]: Also, using `move` as the language tag.

**Review by ronny-mysten — CHANGES_REQUESTED** [review]
Hi @mariobern. It is unadvisable to put Pyth docs on the Sui docs site. This would create two surfaces that would need to capture any updates. It's better to place this info on the Pyth site and link to it from sui docs. I just merged a PR that adds this link: https://docs.sui.io/guides/developer/app-examples/oracle. Let me know if this should point elsewhere or should include more information (like a link to this example).

**Comment by mariobern** [comment]
hey @ronny-mysten thank you, i think this works for now

will try to think of something else to add but in the meantime, i will close this PR

**Pull request closed without merging**